source/common/http/filter_manager.h

```diff
@@ -85,7 +85,8 @@ struct ActiveStreamFilterBase : public virtual StreamFilterCallbacks,
   // Http::StreamFilterCallbacks
   OptRef<const Network::Connection> connection() override;
   Event::Dispatcher& dispatcher() override;
-  void resetStream() override;
+  void resetStream(Http::StreamResetReason reset_reason,
+                   absl::string_view transport_failure_reason) override;
```

Comment:
Out of curiosity, what's the thinking behind some of these methods having default arguments, and some not?

Reply:
just trying to avoid folks using the old constructor from having to change their arguments